### PR TITLE
templates: change additional kubelet service files with new log-level

### PR DESCRIPTION
**Pull request by sjenning**
@rphillips 

https://github.com/openshift/machine-config-operator/pull/2211 did not change the log level for all platforms

### CODE REVIEW

File: templates/master/01-master-kubelet/baremetal/units/kubelet.service.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: kubelet.service
```
Comment (rphillips) [inline templates/master/01-master-kubelet/baremetal/units/kubelet.service.yaml:1]: adding a baremetal config?

**Comment by cgwalters** [comment]
Ah yep, we still really need to de-dup the templates.  Probably handle these explicitly in the code.

File: templates/master/01-master-kubelet/on-prem/units/kubelet.service.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ contents: |
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
   ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
-  Environment="KUBELET_LOG_LEVEL=4"
+  Environment="KUBELET_LOG_LEVEL=3"
```
Comment (darkmuggle) [inline templates/master/01-master-kubelet/on-prem/units/kubelet.service.yaml:13]: Honest question: why not move this its own dropin to `/etc/systemd/system/kubelet.service.d/kubelog.conf` with
```
[Unit]
Environment="KUBELET_LOG_LEVEL=3"
```
That would set the log level in _one_ stop.

**Comment by sjenning** [comment]
/hold
might do what Ben suggests instead.  @rphillips can you do that?

**Comment by rphillips** [comment]
yep... on it

**Comment by sjenning** [comment]
close in favor of https://github.com/openshift/machine-config-operator/pull/2262

**Pull request closed without merging**